doc/source/conf.py

```diff
@@ -54,8 +54,14 @@
 # The short X.Y version.
 version = '5.7'
 # The full version, including alpha/beta/rc tags.
+<<<<<<< HEAD
```

Comment:
Should these merge conflicts still be here?

Reply:
Great catch @kennt-percona, thank you!